test/unit/actions/rotate.test.js

```diff
@@ -0,0 +1,40 @@
+import { describe, it } from 'node:test';
+import { strict as assert } from 'node:assert';
+import * as Rapid from '../../../modules/headless.js';
+
+
+
+// This discrepancy is due to the precision of JavaScript's floating-point arithmetic.
+// The value 6.123233995736766e-17 is extremely close to 0, but it's not exactly 0 due to the limitations
+// of floating-point precision.
+// Helper function to check if two numbers are approximately equal
+function approxEqual(a, b, epsilon = 1e-6) {
```

Comment:
Good catch!  I've started doing this whenever I need a "close to" assert:

https://github.com/rapideditor/rapid-sdk/blob/c7618a4fa5e02e5d1421178f19828946830a746e/packages/math/test/Transform.test.js#L6-L10